diff -ruN influxdb-2.0.8/env influxdb-2.0.8.patch/env
--- influxdb-2.0.8/env	2021-08-13 19:55:41.000000000 +0200
+++ influxdb-2.0.8.patch/env	2021-08-18 05:34:00.993508538 +0200
@@ -14,7 +14,8 @@
 done
 shift $((OPTIND-1))
 
-export PKG_CONFIG="${DIR}/scripts/pkg-config.sh"
+#export PKG_CONFIG="${DIR}/scripts/pkg-config.sh"
+export PKG_CONFIG="${DIR}/bin/linux/pkg-config"
 
 # If this script is being executed, it will be executed under bash
 # so the bash source variable should be present. If the variable
diff -ruN influxdb-2.0.8/Makefile influxdb-2.0.8.patch/Makefile
--- influxdb-2.0.8/Makefile	2021-08-13 19:55:41.000000000 +0200
+++ influxdb-2.0.8.patch/Makefile	2021-08-18 05:34:54.751997548 +0200
@@ -26,12 +26,12 @@
 GO_TEST_ARGS := -tags '$(GO_TEST_TAGS)'
 GO_BUILD_ARGS := -tags '$(GO_BUILD_TAGS)'
 
-ifeq ($(OS), Windows_NT)
-	VERSION := $(shell git describe --exact-match --tags 2>nil)
-else
-	VERSION := $(shell git describe --exact-match --tags 2>/dev/null)
-endif
-COMMIT := $(shell git rev-parse --short HEAD)
+#ifeq ($(OS), Windows_NT)
+#	VERSION := $(shell git describe --exact-match --tags 2>nil)
+#else
+#	VERSION := $(shell git describe --exact-match --tags 2>/dev/null)
+#endif
+#COMMIT := $(shell git rev-parse --short HEAD)
 
 LDFLAGS := $(LDFLAGS) -X main.commit=$(COMMIT)
 ifdef VERSION
@@ -44,7 +44,8 @@
 GO_TEST_PATHS=./...
 
 # Test vars can be used by all recursive Makefiles
-export PKG_CONFIG:=$(PWD)/scripts/pkg-config.sh
+#export PKG_CONFIG:=$(PWD)/scripts/pkg-config.sh
+export PKG_CONFIG:=${PWD}/bin/linux/pkg-config
 export GO_BUILD=env GO111MODULE=on go build $(GO_BUILD_ARGS) -ldflags "$(LDFLAGS)"
 export GO_BUILD_SM=env GO111MODULE=on go build $(GO_BUILD_ARGS) -ldflags "-s -w $(LDFLAGS)"
 export GO_INSTALL=env GO111MODULE=on go install $(GO_BUILD_ARGS) -ldflags "$(LDFLAGS)"
@@ -53,7 +54,7 @@
 export GO_GENERATE=go generate $(GO_BUILD_ARGS)
 export GO_VET=env GO111MODULE=on go vet $(GO_TEST_ARGS)
 export GO_RUN=env GO111MODULE=on go run $(GO_BUILD_ARGS)
-export PATH := $(PWD)/bin/$(GOOS):$(PATH)
+export PATH := $(PWD)/bin/$(GOOS):$(GOPATH)/bin:$(PATH)
 
 
 # All go source files
diff -ruN influxdb-2.0.8/ui/Makefile influxdb-2.0.8.patch/ui/Makefile
--- influxdb-2.0.8/ui/Makefile	2021-08-13 19:55:41.000000000 +0200
+++ influxdb-2.0.8.patch/ui/Makefile	2021-08-18 05:34:01.006842323 +0200
@@ -6,7 +6,7 @@
 	yarn generate
 
 node_modules:
-	yarn install
+	yarn install $(YARN_INSTALL_OPTIONS)
 
 e2e: node_modules
 	yarn generate
diff -ruN influxdb-2.0.8/ui/package.json influxdb-2.0.8.patch/ui/package.json
--- influxdb-2.0.8/ui/package.json	2021-08-13 19:55:41.000000000 +0200
+++ influxdb-2.0.8.patch/ui/package.json	2021-08-18 05:34:01.006842323 +0200
@@ -22,8 +22,8 @@
     "start:dev-cloud": "cross-env CLOUD_LOGOUT_URL=http://localhost:8080/api/v2/signout CLOUD_URL=http://localhost:4000 webpack-dev-server --config ./webpack.dev.ts",
     "start:docker": "yarn generate && yarn build:vendor && yarn run start:dev",
     "build": "yarn install --silent && yarn build:ci",
-    "build:ci": "yarn generate && yarn build:vendor && webpack --config webpack.prod.ts --bail",
-    "build:vendor": "webpack --config webpack.vendor.ts",
+    "build:ci": "yarn generate && yarn build:vendor && webpack --config webpack.prod.ts --bail --progress false",
+    "build:vendor": "webpack --config webpack.vendor.ts --progress false",
     "clean": "rm -rf ./build && rm -rf ./.cache && rm -rf node_modules && rm -rf cypress/screenshots && rm -rf cypress/videos && rm -f junit-results/* ",
     "test": "jest --maxWorkers=2",
     "test:watch": "jest --watch --verbose false",
