diff -ruN influxdb-2.0.3/env influxdb-2.0.3.patch/env
--- influxdb-2.0.3/env	2020-12-15 01:12:33.000000000 +0100
+++ influxdb-2.0.3.patch/env	2020-12-16 05:08:41.851777187 +0100
@@ -14,7 +14,8 @@
 done
 shift $((OPTIND-1))
 
-export PKG_CONFIG="${DIR}/scripts/pkg-config.sh"
+#export PKG_CONFIG="${DIR}/scripts/pkg-config.sh"
+export PKG_CONFIG="${DIR}/bin/linux/pkg-config"
 
 # If this script is being executed, it will be executed under bash
 # so the bash source variable should be present. If the variable
diff -ruN influxdb-2.0.3/Makefile influxdb-2.0.3.patch/Makefile
--- influxdb-2.0.3/Makefile	2020-12-15 01:12:33.000000000 +0100
+++ influxdb-2.0.3.patch/Makefile	2020-12-16 05:09:57.771004292 +0100
@@ -32,12 +32,12 @@
 GO_TEST_ARGS := -tags '$(GO_TEST_TAGS)'
 GO_BUILD_ARGS := -tags '$(GO_BUILD_TAGS)'
 
-ifeq ($(OS), Windows_NT)
-	VERSION := $(shell git describe --exact-match --tags 2>nil)
-else
-	VERSION := $(shell git describe --exact-match --tags 2>/dev/null)
-endif
-COMMIT := $(shell git rev-parse --short HEAD)
+#ifeq ($(OS), Windows_NT)
+#	VERSION := $(shell git describe --exact-match --tags 2>nil)
+#else
+#	VERSION := $(shell git describe --exact-match --tags 2>/dev/null)
+#endif
+#COMMIT := $(shell git rev-parse --short HEAD)
 
 LDFLAGS := $(LDFLAGS) -X main.commit=$(COMMIT)
 ifdef VERSION
@@ -46,7 +46,8 @@
 
 
 # Test vars can be used by all recursive Makefiles
-export PKG_CONFIG:=$(PWD)/scripts/pkg-config.sh
+#export PKG_CONFIG:=$(PWD)/scripts/pkg-config.sh
+export PKG_CONFIG:=${PWD}/bin/linux/pkg-config
 export GO_BUILD=env GO111MODULE=on go build $(GO_BUILD_ARGS) -ldflags "$(LDFLAGS)"
 export GO_BUILD_SM=env GO111MODULE=on go build $(GO_BUILD_ARGS) -ldflags "-s -w $(LDFLAGS)"
 export GO_INSTALL=env GO111MODULE=on go install $(GO_BUILD_ARGS) -ldflags "$(LDFLAGS)"
diff -ruN influxdb-2.0.3/ui/Makefile influxdb-2.0.3.patch/ui/Makefile
--- influxdb-2.0.3/ui/Makefile	2020-12-15 01:12:33.000000000 +0100
+++ influxdb-2.0.3.patch/ui/Makefile	2020-12-16 05:10:34.452241399 +0100
@@ -6,7 +6,7 @@
 	yarn generate
 
 node_modules:
-	yarn install
+	yarn install $(YARN_INSTALL_OPTIONS)
 
 e2e: node_modules
 	yarn generate
diff -ruN influxdb-2.0.3/ui/package.json influxdb-2.0.3.patch/ui/package.json
--- influxdb-2.0.3/ui/package.json	2020-12-15 01:12:33.000000000 +0100
+++ influxdb-2.0.3.patch/ui/package.json	2020-12-16 05:11:54.638278811 +0100
@@ -22,8 +22,8 @@
     "start:dev-cloud": "cross-env CLOUD_LOGOUT_URL=http://localhost:8080/api/v2/signout CLOUD_URL=http://localhost:4000 webpack-dev-server --config ./webpack.dev.ts",
     "start:docker": "yarn generate && yarn build:vendor && yarn run start:dev",
     "build": "yarn install --silent && yarn build:ci",
-    "build:ci": "yarn generate && yarn build:vendor && webpack --config webpack.prod.ts --bail",
-    "build:vendor": "webpack --config webpack.vendor.ts",
+    "build:ci": "yarn generate && yarn build:vendor && webpack --config webpack.prod.ts --bail --progress false",
+    "build:vendor": "webpack --config webpack.vendor.ts --progress false",
     "clean": "rm -rf ./build && rm -rf ./.cache && rm -rf node_modules && rm -rf cypress/screenshots && rm -rf cypress/videos && rm -f junit-results/* ",
     "test": "jest --maxWorkers=2",
     "test:watch": "jest --watch --verbose false",
