diff -ruN influxdb-2.0.1/env influxdb-2.0.1.patched/env
--- influxdb-2.0.1/env	2020-11-11 04:04:56.000000000 +0100
+++ influxdb-2.0.1.patched/env	2020-11-15 22:02:44.657828413 +0100
@@ -14,7 +14,8 @@
 done
 shift $((OPTIND-1))
 
-export PKG_CONFIG="${DIR}/scripts/pkg-config.sh"
+#export PKG_CONFIG="${DIR}/scripts/pkg-config.sh"
+export PKG_CONFIG="${DIR}/bin/linux/pkg-config"
 
 # If this script is being executed, it will be executed under bash
 # so the bash source variable should be present. If the variable
diff -ruN influxdb-2.0.1/Makefile influxdb-2.0.1.patched/Makefile
--- influxdb-2.0.1/Makefile	2020-11-11 04:04:56.000000000 +0100
+++ influxdb-2.0.1.patched/Makefile	2020-11-15 22:02:31.144038336 +0100
@@ -16,12 +16,12 @@
 SUBDIRS := http ui chronograf query storage
 GO_TAGS=
 GO_ARGS=-tags '$(GO_TAGS)'
-ifeq ($(OS), Windows_NT)
-	VERSION := $(shell git describe --exact-match --tags 2>nil)
-else
-	VERSION := $(shell git describe --exact-match --tags 2>/dev/null)
-endif
-COMMIT := $(shell git rev-parse --short HEAD)
+#ifeq ($(OS), Windows_NT)
+#	VERSION := $(shell git describe --exact-match --tags 2>nil)
+#else
+#	VERSION := $(shell git describe --exact-match --tags 2>/dev/null)
+#endif
+#COMMIT := $(shell git rev-parse --short HEAD)
 
 LDFLAGS := $(LDFLAGS) -X main.commit=$(COMMIT)
 ifdef VERSION
@@ -30,7 +30,8 @@
 
 
 # Test vars can be used by all recursive Makefiles
-export PKG_CONFIG:=$(PWD)/scripts/pkg-config.sh
+#export PKG_CONFIG:=$(PWD)/scripts/pkg-config.sh
+export PKG_CONFIG:=$(PWD)/bin/linux/pkg-config
 export GOOS=$(shell go env GOOS)
 export GO_BUILD=env GO111MODULE=on go build $(GO_ARGS) -ldflags "$(LDFLAGS)"
 export GO_BUILD_SM=env GO111MODULE=on go build $(GO_ARGS) -ldflags "-s -w $(LDFLAGS)"
diff -ruN influxdb-2.0.1/ui/Makefile influxdb-2.0.1.patched/ui/Makefile
--- influxdb-2.0.1/ui/Makefile	2020-11-11 04:04:56.000000000 +0100
+++ influxdb-2.0.1.patched/ui/Makefile	2020-11-15 20:44:51.606658454 +0100
@@ -6,7 +6,7 @@
 	yarn generate
 
 node_modules:
-	yarn install
+	yarn install $(YARN_INSTALL_OPTIONS)
 
 e2e: node_modules
 	yarn generate
diff -ruN influxdb-2.0.1/ui/package.json influxdb-2.0.1.patched/ui/package.json
--- influxdb-2.0.1/ui/package.json	2020-11-11 04:04:56.000000000 +0100
+++ influxdb-2.0.1.patched/ui/package.json	2020-11-15 21:28:07.567662592 +0100
@@ -22,8 +22,8 @@
     "start:dev-cloud": "cross-env CLOUD_LOGOUT_URL=http://localhost:8080/api/v2/signout CLOUD_URL=http://localhost:4000 webpack-dev-server --config ./webpack.dev.ts",
     "start:docker": "yarn generate && yarn build:vendor && yarn run start:dev",
     "build": "yarn install --silent && yarn build:ci",
-    "build:ci": "yarn generate && yarn build:vendor && webpack --config webpack.prod.ts --bail",
-    "build:vendor": "webpack --config webpack.vendor.ts",
+    "build:ci": "yarn generate && yarn build:vendor && webpack --config webpack.prod.ts --bail --progress false",
+    "build:vendor": "webpack --config webpack.vendor.ts --progress false",
     "clean": "rm -rf ./build && rm -rf ./.cache && rm -rf node_modules && rm -rf cypress/screenshots && rm -rf cypress/videos && rm -f junit-results/* ",
     "test": "jest --maxWorkers=2",
     "test:watch": "jest --watch --verbose false",
