--- src/mod_auth_kerb.c.orig	Thu Dec  4 20:14:03 2008
+++ src/mod_auth_kerb.c	Fri Apr 27 07:55:06 2012
@@ -62,6 +62,12 @@
 #include <http_protocol.h>
 #include <http_request.h>
 
+#ifdef APLOG_USE_MODULE
+APLOG_USE_MODULE(auth_kerb);
+#undef APLOG_MARK
+#define APLOG_MARK	__FILE__, __LINE__
+#endif 
+
 #ifdef STANDARD20_MODULE_STUFF
 #include <apr_strings.h>
 #include <apr_base64.h>
@@ -360,7 +366,11 @@
 
    
 #ifdef STANDARD20_MODULE_STUFF
+#ifdef APLOG_USE_MODULE
+   ap_log_rerror(file, line, APLOG_MODULE_INDEX, level | APLOG_NOERRNO, status, r, "%s", errstr);
+#else
    ap_log_rerror(file, line, level | APLOG_NOERRNO, status, r, "%s", errstr);
+#endif
 #else
    ap_log_rerror(file, line, level | APLOG_NOERRNO, r, "%s", errstr);
 #endif
@@ -1555,7 +1565,7 @@
    char keyname[1024];
 
    snprintf(keyname, sizeof(keyname) - 1,
-	"mod_auth_kerb::connection::%s::%ld", r->connection->remote_ip, 
+	"mod_auth_kerb::connection::%s::%ld", r->useragent_ip,
 	r->connection->id);
 
    if (apr_pool_userdata_get((void**)&conn_data, keyname, r->connection->pool) != 0)
@@ -1709,7 +1719,7 @@
        prevauth->last_return = ret;
        snprintf(keyname, sizeof(keyname) - 1,
            "mod_auth_kerb::connection::%s::%ld", 
-	   r->connection->remote_ip, r->connection->id);
+	   r->useragent_ip, r->connection->id);
        apr_pool_userdata_set(prevauth, keyname, NULL, r->connection->pool);
    }
 
