diff -ruN kea-1.9.7/configure kea-1.9.7.patched/configure
--- kea-1.9.7/configure	2021-04-26 11:42:18.000000000 +0200
+++ kea-1.9.7.patched/configure	2021-05-08 02:53:44.408965997 +0200
@@ -17029,6 +17029,7 @@
         cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 #include <thread>
+#include <memory>
                          std::shared_ptr<std::thread> th;
 int
 main ()
diff -ruN kea-1.9.7/src/lib/database/backend_selector.cc kea-1.9.7.patched/src/lib/database/backend_selector.cc
--- kea-1.9.7/src/lib/database/backend_selector.cc	2021-04-26 11:42:07.000000000 +0200
+++ kea-1.9.7.patched/src/lib/database/backend_selector.cc	2021-05-08 03:09:33.464344759 +0200
@@ -8,7 +8,7 @@
 
 #include <database/backend_selector.h>
 #include <exceptions/exceptions.h>
-#include <climits>
+#include <limits>
 #include <sstream>
 
 using namespace isc::data;
diff -ruN kea-1.9.7/src/lib/dhcpsrv/subnet_id.h kea-1.9.7.patched/src/lib/dhcpsrv/subnet_id.h
--- kea-1.9.7/src/lib/dhcpsrv/subnet_id.h	2021-04-26 11:42:07.000000000 +0200
+++ kea-1.9.7.patched/src/lib/dhcpsrv/subnet_id.h	2021-05-08 03:10:16.105780550 +0200
@@ -10,6 +10,7 @@
 #include <exceptions/exceptions.h>
 #include <stdint.h>
 #include <typeinfo>
+#include <limits>
 
 namespace isc {
 namespace dhcp {
