diff -rupN raspberrypi-userland64-9999/host_applications/linux/apps/raspicam/CMakeLists.txt raspberrypi-userland64-9999.patch/host_applications/linux/apps/raspicam/CMakeLists.txt
--- raspberrypi-userland64-9999/host_applications/linux/apps/raspicam/CMakeLists.txt	2020-06-07 13:33:47.519615009 +0200
+++ raspberrypi-userland64-9999.patch/host_applications/linux/apps/raspicam/CMakeLists.txt	2020-06-07 14:18:54.484369043 +0200
@@ -13,12 +13,12 @@ include_directories(${PROJECT_SOURCE_DIR
 include_directories(${PROJECT_SOURCE_DIR}/host_applications/linux/libs/sm)
 
 # Find the commit hash of the build and pass to the compiler
-execute_process(
-  COMMAND git log -1 --abbrev=12 --format=%h
-  OUTPUT_VARIABLE GIT_COMMIT_ID
-  OUTPUT_STRIP_TRAILING_WHITESPACE
-)
-add_definitions("-DGIT_COMMIT_ID=\"${GIT_COMMIT_ID}\"")
+#execute_process(
+#  COMMAND git log -1 --abbrev=12 --format=%h
+#  OUTPUT_VARIABLE GIT_COMMIT_ID
+#  OUTPUT_STRIP_TRAILING_WHITESPACE
+#)
+add_definitions("-DGIT_COMMIT_ID=\"HEAD\"")
 
 # Determine if we are tainted
 if(NOT SKIP_TAINTED_CHECK)
