diff -rupN freeswitch-1.6.2/configure freeswitch-1.6.2.automake/configure
--- freeswitch-1.6.2/configure	2015-09-28 07:27:47.000000000 +0200
+++ freeswitch-1.6.2.automake/configure	2015-10-11 13:28:00.457090284 +0200
@@ -17902,7 +17902,7 @@ if test "${ac_cv_gcc_supports_w_no_unuse
   $as_echo_n "(cached) " >&6
 else
 
-      CFLAGS="$CFLAGS -Wno-unused-result -Wno-error=unused-result"
+      CFLAGS="$CFLAGS"
       cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
@@ -18341,9 +18341,9 @@ elif test "x${ax_cv_c_compiler_vendor}"
 
   if test "x$SWITCH_AM_CFLAGS" = "x"; then
     test "x$silent" != "xyes" && echo "  setting SWITCH_AM_CFLAGS to \"-Werror\""
-    SWITCH_AM_CFLAGS="-Werror"
+    SWITCH_AM_CFLAGS=""
   else
-    apr_addto_bugger="-Werror"
+    apr_addto_bugger=""
     for i in $apr_addto_bugger; do
       apr_addto_duplicate="0"
       for j in $SWITCH_AM_CFLAGS; do
@@ -18406,9 +18406,9 @@ elif test "x${ax_cv_c_compiler_vendor}"
 
   if test "x$SWITCH_AM_CFLAGS" = "x"; then
     test "x$silent" != "xyes" && echo "  setting SWITCH_AM_CFLAGS to \"-Werror\""
-    SWITCH_AM_CFLAGS="-Werror"
+    SWITCH_AM_CFLAGS=""
   else
-    apr_addto_bugger="-Werror"
+    apr_addto_bugger=""
     for i in $apr_addto_bugger; do
       apr_addto_duplicate="0"
       for j in $SWITCH_AM_CFLAGS; do
@@ -18929,9 +18929,9 @@ esac
 
   if test "x$SWITCH_AM_CFLAGS" = "x"; then
     test "x$silent" != "xyes" && echo "  setting SWITCH_AM_CFLAGS to \"-Wno-unused-result\""
-    SWITCH_AM_CFLAGS="-Wno-unused-result"
+    SWITCH_AM_CFLAGS=""
   else
-    apr_addto_bugger="-Wno-unused-result"
+    apr_addto_bugger=""
     for i in $apr_addto_bugger; do
       apr_addto_duplicate="0"
       for j in $SWITCH_AM_CFLAGS; do
diff -rupN freeswitch-1.6.2/libs/esl/php/Makefile.in freeswitch-1.6.2.automake/libs/esl/php/Makefile.in
--- freeswitch-1.6.2/libs/esl/php/Makefile.in	2015-09-28 07:27:49.000000000 +0200
+++ freeswitch-1.6.2.automake/libs/esl/php/Makefile.in	2015-10-11 13:47:00.887757755 +0200
@@ -573,7 +573,7 @@ uninstall-am:
 all: ESL.so
 
 esl_wrap.cpp:
-	swig2.0 -module ESL -php5 -c++ -DMULTIPLICITY -I../src/include -o esl_wrap.cpp ../ESL.i
+	swig -module ESL -php5 -c++ -DMULTIPLICITY -I../src/include -o esl_wrap.cpp ../ESL.i
 	sed -e 's/  char \*type_name;/  const char \*type_name;/' -i esl_wrap.cpp
 
 esl_wrap.o: esl_wrap.cpp
@@ -595,8 +595,6 @@ install: ESL.so
 	cp ESL.so "$(DESTDIR)$(PHP_EXT_DIR)"
 	test -d "$(DESTDIR)$(PHP_INC_DIR)" || mkdir -p "$(DESTDIR)$(PHP_INC_DIR)"
 	cp ESL.php "$(DESTDIR)$(PHP_INC_DIR)"
-	test -d "$(DESTDIR)$(PHP_INI_DIR)" || mkdir -p "$(DESTDIR)$(PHP_INI_DIR)"
-	test -f "$(DESTDIR)$(PHP_INI_DIR)/esl.ini" || echo 'extension=ESL.so' > "$(DESTDIR)$(PHP_INI_DIR)/esl.ini"
 
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
