diff -rupN freeswitch-1.6.2/Makefile.in freeswitch-1.6.2.minimal-config/Makefile.in
--- freeswitch-1.6.2/Makefile.in	2015-09-28 07:28:06.000000000 +0200
+++ freeswitch-1.6.2.minimal-config/Makefile.in	2015-10-11 12:20:43.766442753 +0200
@@ -3452,7 +3452,7 @@ config-%:
 			$(INSTALL) -m 644 $(switch_srcdir)/conf/$*/$$dir/$$filename $(DESTDIR)$(confdir)$$dir; \
 	done
 
-samples-conf: config-vanilla
+samples-conf: config-minimal
 
 samples-htdocs: 
 	test -d $(DESTDIR)$(htdocsdir) || $(mkinstalldirs) $(DESTDIR)$(htdocsdir)
