Index: svn/src/headerField.cpp
===================================================================
--- svn/src/headerField.cpp	2013-12-04 14:20:38.864837137 +0100
+++ svn/src/headerField.cpp	2013-12-04 14:20:38.844837137 +0100
@@ -132,7 +132,9 @@
 				                  buffer.begin() + nameEnd);

 				// Skip ':' character
-				++pos;
+				// Patch: don't just skip one colon, but any and all, if available. NS 2 December 2013
+				while ((pos < end) && (buffer[pos] == ':'))
+				    ++pos;

 				// Skip spaces between ':' and the field contents
 				while (pos < end && (buffer[pos] == ' ' || buffer[pos] == '\t'))
