diff -Nurb libvmime-0.9.2.orig/src/parameterizedHeaderField.cpp libvmime-0.9.2.patched/src/parameterizedHeaderField.cpp
--- libvmime-0.9.2.orig/src/parameterizedHeaderField.cpp	(revision 603)
+++ libvmime-0.9.2.patched/src/parameterizedHeaderField.cpp	(working copy)
@@ -97,7 +97,7 @@
 	// Advance up to ';', if any
 	string::size_type valueLength = 0;
 
-	while (p < pend && *p != ';')  // FIXME: support ";" inside quoted or RFC-2047-encoded text
+	while (p < pend && *p != ';' && (!parserHelpers::isSpace(*p)))  // FIXME: support ";" inside quoted or RFC-2047-encoded text
 	{
 		++p;
 		++valueLength;
@@ -118,6 +118,10 @@
 	{
 		std::map <string, paramInfo> params;
 
+		if (*p != ';')
+			while (p < pend && *p != ';')  // FIXME: support ";" inside quoted or RFC-2047-encoded text
+				++p;
+
 		while (*p == ';')
 		{
 			// Skip ';'
