diff -rupN zarafa-7.1.8/ECtools/zarafa-archiver/ArchiverSession.cpp zarafa-7.1.8.patched/ECtools/zarafa-archiver/ArchiverSession.cpp
--- zarafa-7.1.8/ECtools/zarafa-archiver/ArchiverSession.cpp	2013-12-20 15:28:14.000000000 +0100
+++ zarafa-7.1.8.patched/ECtools/zarafa-archiver/ArchiverSession.cpp	2014-01-31 22:17:56.194654270 +0100
@@ -55,8 +55,6 @@
 #include "mapiext.h"
 #include "userutil.h"
 #include "ECMsgStore.h"
-#include "ECLicense.h"
-#include "ECMAPILicense.h"
 
 typedef mapi_memory_ptr<ECSERVERLIST> ECServerListPtr;
 
diff -rupN zarafa-7.1.8/ECtools/zarafa-archiver/Makefile.am zarafa-7.1.8.patched/ECtools/zarafa-archiver/Makefile.am
--- zarafa-7.1.8/ECtools/zarafa-archiver/Makefile.am	2013-12-20 14:35:15.000000000 +0100
+++ zarafa-7.1.8.patched/ECtools/zarafa-archiver/Makefile.am	2014-01-31 22:19:23.287453264 +0100
@@ -10,8 +10,6 @@ AM_CPPFLAGS = ${ZCPPFLAGS} $(DEBUGFLAGS)
 	-I${top_srcdir}/provider/include \
 	-I${top_srcdir}/provider/soap \
 	-I${top_builddir}/provider/soap \
-	-I${top_srcdir}/liblicense \
-	-I${top_builddir}/liblicense \
 	-I${top_srcdir}/common \
 	$(MYSQL_INCLUDES)
 
@@ -19,9 +17,7 @@ AM_CPPFLAGS = ${ZCPPFLAGS} $(DEBUGFLAGS)
 
 libarchiver_la_LIBADD = ${top_builddir}/mapi4linux/src/libmapi.la \
 	${top_builddir}/common/libcommon_mapi.la \
-	${top_builddir}/common/libcommon_util.la \
-	${top_builddir}/liblicense/liblicense.la \
-	${top_builddir}/liblicense/liblicense_mapi.la
+	${top_builddir}/common/libcommon_util.la
 
 libarchiver_la_SOURCES = \
 	ArchiverSession.cpp ArchiverSession.h archiver-session_fwd.h \
diff -rupN zarafa-7.1.8/ECtools/zarafa-archiver/Makefile.in zarafa-7.1.8.patched/ECtools/zarafa-archiver/Makefile.in
--- zarafa-7.1.8/ECtools/zarafa-archiver/Makefile.in	2013-12-20 14:35:33.000000000 +0100
+++ zarafa-7.1.8.patched/ECtools/zarafa-archiver/Makefile.in	2014-01-31 22:19:43.241427848 +0100
@@ -89,9 +89,7 @@ am__v_lt_0 = --silent
 libarchiver_la_DEPENDENCIES =  \
 	${top_builddir}/mapi4linux/src/libmapi.la \
 	${top_builddir}/common/libcommon_mapi.la \
-	${top_builddir}/common/libcommon_util.la \
-	${top_builddir}/liblicense/liblicense.la \
-	${top_builddir}/liblicense/liblicense_mapi.la
+	${top_builddir}/common/libcommon_util.la
 am_libarchiver_la_OBJECTS = ArchiverSession.lo archiver-common.lo \
 	ArchiveManageImpl.lo ArchiveStateCollector.lo \
 	ArchiveStateUpdater.lo ArchiveHelper.lo storehelper.lo \
@@ -364,16 +362,12 @@ AM_CPPFLAGS = ${ZCPPFLAGS} $(DEBUGFLAGS)
 	-I${top_srcdir}/provider/include \
 	-I${top_srcdir}/provider/soap \
 	-I${top_builddir}/provider/soap \
-	-I${top_srcdir}/liblicense \
-	-I${top_builddir}/liblicense \
 	-I${top_srcdir}/common \
 	$(MYSQL_INCLUDES)
 
 libarchiver_la_LIBADD = ${top_builddir}/mapi4linux/src/libmapi.la \
 	${top_builddir}/common/libcommon_mapi.la \
-	${top_builddir}/common/libcommon_util.la \
-	${top_builddir}/liblicense/liblicense.la \
-	${top_builddir}/liblicense/liblicense_mapi.la
+	${top_builddir}/common/libcommon_util.la
 
 libarchiver_la_SOURCES = \
 	ArchiverSession.cpp ArchiverSession.h archiver-session_fwd.h \
diff -rupN zarafa-7.1.8/php-ext/typeconversion.cpp zarafa-7.1.8.patched/php-ext/typeconversion.cpp
--- zarafa-7.1.8/php-ext/typeconversion.cpp	2013-12-20 15:28:14.000000000 +0100
+++ zarafa-7.1.8.patched/php-ext/typeconversion.cpp	2014-01-31 22:17:27.273724756 +0100
@@ -65,6 +65,9 @@ extern "C" {
    	#include "php_globals.h"
 	#include "ext/standard/info.h"
 	#include "ext/standard/php_string.h"
+#ifdef ZTS
+	void*** tsrm_ls;
+#endif
 }
 
 #undef inline
