diff -rupN zcp-7.2.3.602/swig/python/zarafa/zarafa/__init__.py zcp-7.2.3.602.logging/swig/python/zarafa/zarafa/__init__.py
--- zcp-7.2.3.602/swig/python/zarafa/zarafa/__init__.py	2016-04-08 15:50:39.000000000 +0200
+++ zcp-7.2.3.602.logging/swig/python/zarafa/zarafa/__init__.py	2016-04-09 19:09:09.073483903 +0200
@@ -3858,7 +3858,6 @@ def logger(service, options=None, stdout
     logger = logging.getLogger(name or service)
     if logger.handlers:
         return logger
-    formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
     log_method = 'file'
     log_file = '/var/log/zarafa/%s.log' % service
     if config:
@@ -3868,10 +3867,13 @@ def logger(service, options=None, stdout
     if name:
         log_file = log_file.replace(service, name) # XXX
     fh = None
+    formatter = None
     if log_method == 'file' and log_file != '-':
         fh = logging.handlers.WatchedFileHandler(log_file)
+        formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
     elif log_method == 'syslog':
         fh = logging.handlers.SysLogHandler(address='/dev/log')
+        formatter = logging.Formatter('zarafa-search: %(message)s')
     if fh:
         fh.setLevel(log_level)
         fh.setFormatter(formatter)
