diff -rupN zarafa-7.1.8/configure.ac zarafa-7.1.8.automake/configure.ac
--- zarafa-7.1.8/configure.ac	2014-01-30 12:38:14.000000000 +0100
+++ zarafa-7.1.8.automake/configure.ac	2014-02-01 10:02:04.299170143 +0100
@@ -812,21 +812,6 @@ AC_CONFIG_FILES([Makefile
 		version
 		specialbuild
 ])
-dnl non-oss files
-if test -d `dirname $0`/licensed; then
-AC_CONFIG_FILES([
-		liblicense/Makefile
-		licensed/Makefile
-		ECtools/zarafa-backup/Makefile
-		ECtools/zarafa-backup/helpers/Makefile
-		ECtools/zarafa-report/Makefile
-		ECtools/zarafa-msr/Makefile
-		ECtools/zarafa-ocf/Makefile
-		ECtools/za-extra/Makefile
-		ECtools/zarafa-ws/Makefile
-		ECtools/zarafa-ws/etc/Makefile
-])
-fi
 
 AC_OUTPUT
 
diff -rupN zarafa-7.1.8/ECtools/Makefile.am zarafa-7.1.8.automake/ECtools/Makefile.am
--- zarafa-7.1.8/ECtools/Makefile.am	2014-01-30 12:44:38.000000000 +0100
+++ zarafa-7.1.8.automake/ECtools/Makefile.am	2014-02-01 09:56:06.493524301 +0100
@@ -1,5 +1 @@
-if ! OSS_ONLY
-PROSUBS = zarafa-backup zarafa-report zarafa-msr za-extra zarafa-ocf zarafa-ws
-endif
-
-SUBDIRS = $(PROSUBS) zarafa-archiver zarafa-admin zarafa-monitor zarafa-passwd zarafa-fsck zarafa-cfgchecker zarafa-stats zarafa-search zarafa-utils
+SUBDIRS = zarafa-archiver zarafa-admin zarafa-monitor zarafa-passwd zarafa-fsck zarafa-cfgchecker zarafa-stats zarafa-search zarafa-utils
diff -rupN zarafa-7.1.8/Makefile.am zarafa-7.1.8.automake/Makefile.am
--- zarafa-7.1.8/Makefile.am	2014-01-30 12:44:40.000000000 +0100
+++ zarafa-7.1.8.automake/Makefile.am	2014-02-01 10:07:30.669229630 +0100
@@ -1,10 +1,6 @@
 ACLOCAL_AMFLAGS = -I autoconf
 
-if ! OSS_ONLY
-PROSUBS = liblicense licensed
-endif
-
-SUBDIRS = common libfreebusy mapi4linux zarafa-libsync $(PROSUBS) provider libicalmapi inetmapi
+SUBDIRS = common libfreebusy mapi4linux zarafa-libsync provider libicalmapi inetmapi
 if WITH_PHP
 SUBDIRS += php-ext
 endif
