diff -rupN zcp-7.2.3.602/php-ext/Makefile.am zcp-7.2.3.602.php/php-ext/Makefile.am
--- zcp-7.2.3.602/php-ext/Makefile.am	2016-04-08 15:50:39.000000000 +0200
+++ zcp-7.2.3.602.php/php-ext/Makefile.am	2016-04-09 11:32:47.083685389 +0200
@@ -38,7 +38,7 @@ datadir = ${datarootdir}/php/mapi
 # to only apache2-mod_php5 and not php-cli, php-fpm, etc.
 #
 sysconfdir = ${PHP_SYSCONF_DIR}
-dist_sysconf_DATA = zarafa.ini
+dist_sysconf_DATA = 
 
 sampleconfdir = ${docdir}/example-config
 sampleconf_DATA = zarafa.ini
