diff -rupN zcp-7.2.3.602/installer/linux/Makefile.am zcp-7.2.3.602.sysconfig/installer/linux/Makefile.am
--- zcp-7.2.3.602/installer/linux/Makefile.am	2016-04-08 15:50:39.000000000 +0200
+++ zcp-7.2.3.602.sysconfig/installer/linux/Makefile.am	2016-04-09 08:27:19.420073056 +0200
@@ -53,7 +53,7 @@ suse_sysconfig_path = ${localstatedir}/a
 sles_sysconfig_path = ${localstatedir}/adm/fillup-templates
 sysconf_sysconfigdir = ${@DISTRO@_sysconfig_path}
 sysconf_sysconfig_DATA = sysconfig.txt
-alt_sysconfig_path = ${sysconfdir}/sysconfig
+alt_sysconfig_path = ${sysconfdir}/default
 
 dist_doc_DATA = \
 	db-convert-4.1-to-4.2 db-convert-4.20-to-4.21 db-convert-4.20-to-innodb.sql db-calc-storesize db-convert-attachments-to-files \
