diff -rupN zcp-7.2.3.602/ECtools/search/zarafa_search/__init__.py zcp-7.2.3.602.bsddb3/ECtools/search/zarafa_search/__init__.py
--- zcp-7.2.3.602/ECtools/search/zarafa_search/__init__.py	2016-04-08 15:50:39.000000000 +0200
+++ zcp-7.2.3.602.bsddb3/ECtools/search/zarafa_search/__init__.py	2016-04-09 14:47:05.189409151 +0200
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 from contextlib import closing
-import dbhash
+from bsddb3 import db
 import fcntl
 import os.path
 from Queue import Empty
@@ -70,15 +70,17 @@ CONFIG = {
 
 def db_get(db_path, key):
     """ get value from db file """
-    with closing(dbhash.open(db_path, 'c')) as db:
-        return db.get(key)
+    with closing(db.DB()) as keyDB:
+         keyDB.open(db_path, None, db.DB_HASH, db.DB_CREATE)
+         return keyDB.get(key)
 
 def db_put(db_path, key, value):
     """ store key, value in db file """
     with open(db_path+'.lock', 'w') as lockfile:
         fcntl.flock(lockfile.fileno(), fcntl.LOCK_EX)
-        with closing(dbhash.open(db_path, 'c')) as db:
-            db[key] = value
+        with closing(db.DB()) as keyDB:
+            keyDB.open(db_path, None, db.DB_HASH, db.DB_CREATE)
+            keyDB.put(key, value)
 
 class SearchWorker(zarafa.Worker):
     """ process which handles search requests coming from outlook/webapp, according to our internal protocol """
