diff -rupN zarafa-7.1.8/php-ext/typeconversion.cpp zarafa-7.1.8.php/php-ext/typeconversion.cpp
--- zarafa-7.1.8/php-ext/typeconversion.cpp	2014-01-30 13:41:33.000000000 +0100
+++ zarafa-7.1.8.php/php-ext/typeconversion.cpp	2014-02-01 10:27:50.731861657 +0100
@@ -65,6 +65,9 @@ extern "C" {
    	#include "php_globals.h"
 	#include "ext/standard/info.h"
 	#include "ext/standard/php_string.h"
+#ifdef ZTS
+	void*** tsrm_ls;
+#endif
 }
 
 #undef inline
