diff -rupN zarafa-7.1.11/configure.ac zarafa-7.1.11.automake/configure.ac
--- zarafa-7.1.11/configure.ac	2014-09-19 10:01:58.000000000 +0200
+++ zarafa-7.1.11.automake/configure.ac	2014-11-04 20:52:22.969134056 +0100
@@ -835,21 +835,6 @@ AC_CONFIG_FILES([Makefile
 		version
 		specialbuild
 ])
-dnl non-oss files
-if test -d `dirname $0`/licensed; then
-AC_CONFIG_FILES([
-		liblicense/Makefile
-		licensed/Makefile
-		ECtools/zarafa-backup/Makefile
-		ECtools/zarafa-backup/helpers/Makefile
-		ECtools/zarafa-report/Makefile
-		ECtools/zarafa-msr/Makefile
-		ECtools/zarafa-ocf/Makefile
-		ECtools/za-extra/Makefile
-		ECtools/zarafa-ws/Makefile
-		ECtools/zarafa-ws/etc/Makefile
-])
-fi
 
 AC_OUTPUT
 
diff -rupN zarafa-7.1.11/ECtools/Makefile.am zarafa-7.1.11.automake/ECtools/Makefile.am
--- zarafa-7.1.11/ECtools/Makefile.am	2014-09-03 09:55:51.000000000 +0200
+++ zarafa-7.1.11.automake/ECtools/Makefile.am	2014-11-04 20:51:58.568328664 +0100
@@ -1,5 +1 @@
-if ! OSS_ONLY
-PROSUBS = zarafa-backup zarafa-report zarafa-msr za-extra zarafa-ocf zarafa-ws
-endif
-
-SUBDIRS = $(PROSUBS) zarafa-archiver zarafa-admin zarafa-monitor zarafa-passwd zarafa-fsck zarafa-cfgchecker zarafa-stats zarafa-search zarafa-utils
+SUBDIRS = zarafa-archiver zarafa-admin zarafa-monitor zarafa-passwd zarafa-fsck zarafa-cfgchecker zarafa-stats zarafa-search zarafa-utils
diff -rupN zarafa-7.1.11/Makefile.am zarafa-7.1.11.automake/Makefile.am
--- zarafa-7.1.11/Makefile.am	2014-09-19 10:02:13.000000000 +0200
+++ zarafa-7.1.11.automake/Makefile.am	2014-11-04 20:51:45.667902861 +0100
@@ -1,10 +1,6 @@
 ACLOCAL_AMFLAGS = -I autoconf
 
-if ! OSS_ONLY
-PROSUBS = liblicense licensed
-endif
-
-SUBDIRS = external common libfreebusy mapi4linux zarafa-libsync $(PROSUBS) provider libicalmapi inetmapi
+SUBDIRS = external common libfreebusy mapi4linux zarafa-libsync provider libicalmapi inetmapi
 if WITH_PHP
 SUBDIRS += php-ext
 endif
@@ -25,15 +21,9 @@ BUILT_SOURCES = common/ecversion.h
 common/ecversion.h: versionheader.sh
 	sh ${top_srcdir}/versionheader.sh > common/ecversion.h
 
-if ! OSS_ONLY
-CLEANFILES=common/ecversion.h
-endif
-
 dist-hook:
 	find $(distdir) -type d -name .svn -exec rm -rf {} \; 2>/dev/null || true
-if OSS_ONLY
 	[ -x $(top_srcdir)/fixup-distro.sh ] && $(top_srcdir)/fixup-distro.sh $(distdir)
-endif
 
 install-ajax-webaccess:
 	install -d -m 755 $(DESTDIR)/usr/share
