diff -rupN zarafa-7.2.0/common/Makefile.am zarafa-7.2.0.shared/common/Makefile.am
--- zarafa-7.2.0/common/Makefile.am	2014-12-05 09:00:32.000000000 +0100
+++ zarafa-7.2.0.shared/common/Makefile.am	2015-01-12 06:52:36.577880729 +0100
@@ -34,7 +34,7 @@ AM_CPPFLAGS = ${ZCPPFLAGS} $(DEBUGFLAGS)
 	-I${top_srcdir}/mapi4linux/include -I${top_srcdir}/provider/common \
 	-I${top_srcdir}/libfreebusy $(MYSQL_INCLUDES)
 
-libcommon_mapi_la_LDFLAGS = ${AM_LDFLAGS} -static
+libcommon_mapi_la_LDFLAGS = ${AM_LDFLAGS} -release ${PACKAGE_VERSION}
 libcommon_mapi_la_SOURCES = CommonUtil.cpp  \
 			rtfutil.cpp					    \
 			codepage.cpp					\
@@ -58,8 +58,9 @@ libcommon_mapi_la_SOURCES = CommonUtil.c
 			MAPIConsoleTable.cpp			\
 			ECMAPIDebug.cpp					\
 			MAPIErrors.cpp
+libcommon_mapi_la_LIBADD = -lpthread ${ICU_LIBS}
 
-libcommon_util_la_LDFLAGS = ${AM_LDFLAGS} -static
+libcommon_util_la_LDFLAGS = ${AM_LDFLAGS} -release ${PACKAGE_VERSION}
 libcommon_util_la_SOURCES = platform.cpp			\
 			platform.linux.cpp						\
 			md5.cpp									\
@@ -93,19 +94,23 @@ libcommon_util_la_SOURCES = platform.cpp
 			fileutil.cpp							\
 			ECThreadPool.cpp						\
 			ConsoleTable.cpp
+libcommon_util_la_LIBADD = -lpthread ${BOOST_FILESYSTEM_LIBS} ${BOOST_SYSTEM_LIBS}
 
 if HAVE_ICU
 libcommon_util_la_SOURCES += ustringutil/utfutil.cpp
+libcommon_util_la_LIBADD += ${ICU_LIBS}
 endif
 
-libcommon_ssl_la_LDFLAGS = ${AM_LDFLAGS} -static
+libcommon_ssl_la_LDFLAGS = ${AM_LDFLAGS} -release ${PACKAGE_VERSION}
 libcommon_ssl_la_SOURCES = 							\
 			SSLUtil.cpp								\
 			ECChannelClient.cpp						\
 			ECChannel.cpp
+libcommon_ssl_la_LIBADD = -lpthread ${SSL_LIBS}
 
-libcommon_service_la_LDFLAGS = ${AM_LDFLAGS} -static
+libcommon_service_la_LDFLAGS = ${AM_LDFLAGS} -release ${PACKAGE_VERSION}
 libcommon_service_la_SOURCES = ECDatabaseMySQL.cpp
+libcommon_service_la_LIBADD = ${MYSQL_LIBS}
 
 # remove double installed headers
 install-data-hook:
diff -rupN zarafa-7.2.0/common/zarafa.pc.in zarafa-7.2.0.shared/common/zarafa.pc.in
--- zarafa-7.2.0/common/zarafa.pc.in	2014-12-05 09:00:32.000000000 +0100
+++ zarafa-7.2.0.shared/common/zarafa.pc.in	2015-01-11 21:47:12.774523852 +0100
@@ -7,5 +7,5 @@ Name: zarafa
 Description: Zarafa custom development
 Version: @ZARAFA_VERSION@
 Requires:
-Libs: -L${libdir} -lmapi @PROG_LIBS@
+Libs: -L${libdir} -lmapi-@PACKAGE_VERSION@ @PROG_LIBS@
 Cflags: -I${includedir}/mapi4linux/ -I${includedir}/zarafa/ -DUNICODE
diff -rupN zarafa-7.2.0/ECtools/zarafa-archiver/Makefile.am zarafa-7.2.0.shared/ECtools/zarafa-archiver/Makefile.am
--- zarafa-7.2.0/ECtools/zarafa-archiver/Makefile.am	2014-12-05 09:00:29.000000000 +0100
+++ zarafa-7.2.0.shared/ECtools/zarafa-archiver/Makefile.am	2015-01-11 21:41:38.067072625 +0100
@@ -16,6 +16,7 @@ AM_CPPFLAGS = ${ZCPPFLAGS} $(DEBUGFLAGS)
 
 
 
+libarchiver_la_LDFLAGS = -release ${PACKAGE_VERSION}
 libarchiver_la_LIBADD = ${top_builddir}/mapi4linux/src/libmapi.la \
 	${top_builddir}/common/libcommon_mapi.la \
 	${top_builddir}/common/libcommon_util.la
@@ -53,6 +54,7 @@ libarchiver_core_la_SOURCES = \
 	ArchiveControlImpl.cpp ArchiveControlImpl.h \
 	ECDatabase.h ECDatabase.cpp
 
+libarchiver_core_la_LDFLAGS = -release ${PACKAGE_VERSION}
 
 
 zarafa_archiver_LDADD = ${top_builddir}/mapi4linux/src/libmapi.la \
diff -rupN zarafa-7.2.0/libfreebusy/Makefile.am zarafa-7.2.0.shared/libfreebusy/Makefile.am
--- zarafa-7.2.0/libfreebusy/Makefile.am	2014-12-05 09:00:29.000000000 +0100
+++ zarafa-7.2.0.shared/libfreebusy/Makefile.am	2015-01-11 22:12:13.842911446 +0100
@@ -6,7 +6,7 @@ AM_CPPFLAGS = ${ZCPPFLAGS} $(DEBUGFLAGS)
 
 libfreebusy_la_includedir = $(includedir)/libfreebusy
 
-libfreebusy_la_LDFLAGS = ${AM_LDFLAGS} -static
+libfreebusy_la_LDFLAGS = ${AM_LDFLAGS} -release ${PACKAGE_VERSION}
 
 libfreebusy_la_include_HEADERS = ECEnumFBBlock.h ECFBBlockList.h		\
 	ECFreeBusyData.h ECFreeBusySupport.h ECFreeBusyUpdate.h freebusy.h	\
diff -rupN zarafa-7.2.0/mapi4linux/src/Makefile.am zarafa-7.2.0.shared/mapi4linux/src/Makefile.am
--- zarafa-7.2.0/mapi4linux/src/Makefile.am	2014-12-05 09:00:32.000000000 +0100
+++ zarafa-7.2.0.shared/mapi4linux/src/Makefile.am	2015-01-11 21:49:24.178759680 +0100
@@ -4,7 +4,7 @@ AM_CPPFLAGS = ${ZCPPFLAGS} $(DEBUGFLAGS)
 	-DMAPICONFIGDIR=\"${MAPICONFIGDIR}\" \
 	-I$(top_srcdir)/mapi4linux/include -I$(top_srcdir)/provider/common -I${top_srcdir}/common -I${top_srcdir}/libfreebusy
 
-libmapi_la_LDFLAGS = ${AM_LDFLAGS} $(BOOST_FILESYSTEM_LDFLAGS)
+libmapi_la_LDFLAGS = ${AM_LDFLAGS} $(BOOST_FILESYSTEM_LDFLAGS) -release ${PACKAGE_VERSION}
 
 libmapi_la_LIBADD = \
 			${top_builddir}/common/libcommon_mapi.la \
diff -rupN zarafa-7.2.0/provider/common/Makefile.am zarafa-7.2.0.shared/provider/common/Makefile.am
--- zarafa-7.2.0/provider/common/Makefile.am	2014-12-05 09:00:27.000000000 +0100
+++ zarafa-7.2.0.shared/provider/common/Makefile.am	2015-01-12 06:50:44.990948576 +0100
@@ -1,10 +1,11 @@
-noinst_LTLIBRARIES = libzarafacommon.la
+pkglib_LTLIBRARIES = libzarafacommon.la
 
 AM_CPPFLAGS = ${ZCPPFLAGS} $(DEBUGFLAGS) $(GSOAP_CFLAGS) \
 	$(BOOST_CPPFLAGS) \
 	-I${top_srcdir}/mapi4linux/include -I${top_srcdir}/provider/include \
 	-I${top_builddir}/provider/soap -I${top_srcdir}/provider/soap -I${top_srcdir}/common 
 
+libzarafacommon_la_LDFLAGS = -release ${PACKAGE_VERSION}
 libzarafacommon_la_SOURCES = SOAPUtils.cpp SOAPUtils.h \
 	SOAPAlloc.h \
 	SOAPHelpers.cpp SOAPHelpers.h \
@@ -17,6 +18,8 @@ libzarafacommon_la_SOURCES = SOAPUtils.c
 	ECOfflineState.h ECOfflineState.cpp \
 	StreamTypes.h ZarafaVersions.h
 
+libzarafacommon_la_LIBADD = ../soap/libsoapclient.la
+
 check-syntax:
 	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(AM_CXXFLAGS) $(CXXFLAGS) \
 		-fsyntax-only -fmessage-length=0 ${CHK_SOURCES} -Wall -Wformat=2
diff -rupN zarafa-7.2.0/provider/libserver/Makefile.am zarafa-7.2.0.shared/provider/libserver/Makefile.am
--- zarafa-7.2.0/provider/libserver/Makefile.am	2014-12-05 09:00:27.000000000 +0100
+++ zarafa-7.2.0.shared/provider/libserver/Makefile.am	2015-01-12 06:53:40.923289178 +0100
@@ -1,4 +1,4 @@
-noinst_LTLIBRARIES = libserver.la
+pkglib_LTLIBRARIES = libserver.la
 
 AM_CPPFLAGS = ${ZCPPFLAGS} $(DEBUGFLAGS) $(BOOST_CPPFLAGS) \
 	-I${top_srcdir}/mapi4linux/include \
@@ -9,6 +9,7 @@ AM_CPPFLAGS = ${ZCPPFLAGS} $(DEBUGFLAGS)
 	-I${top_srcdir}/common -DPKGLIBDIR='"${pkglibdir}"' \
 	$(MYSQL_INCLUDES) $(GSOAP_CFLAGS) $(SSL_CFLAGS) $(SSL_HAS_EVP_PKEY_CMP) $(KRB5_CFLAGS)
 
+libserver_la_LDFLAGS = -release ${PACKAGE_VERSION}
 libserver_la_SOURCES = \
 	ZarafaCmd.cpp \
 	ECABObjectTable.cpp ECABObjectTable.h \
@@ -61,6 +62,8 @@ libserver_la_SOURCES = \
 	ZarafaCmdUtil.cpp ZarafaCmdUtil.h \
 	ECLockManager.cpp ECLockManager.h
 
+libserver_la_LIBADD = -ldl -lpthread -lz ${SSL_LIBS} ${BOOST_FILESYSTEM_LIBS} ${BOOST_SYSTEM_LIBS} ${ICU_LIBS} ${KRB5_LIBS} ${MYSQL_LIBS} ${PAM_LIBS}
+
 check-syntax:
 	$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(AM_CXXFLAGS) $(CXXFLAGS) \
 		-fsyntax-only -fmessage-length=0 ${CHK_SOURCES} -Wall -Wformat=2
diff -rupN zarafa-7.2.0/provider/server/Makefile.am zarafa-7.2.0.shared/provider/server/Makefile.am
--- zarafa-7.2.0/provider/server/Makefile.am	2014-12-05 09:00:27.000000000 +0100
+++ zarafa-7.2.0.shared/provider/server/Makefile.am	2015-01-12 06:54:20.677904545 +0100
@@ -18,9 +18,10 @@ zarafa_server_LDADD = \
 	${top_builddir}/common/libcommon_util.la								\
 	${top_builddir}/common/libcommon_mapi.la								\
 	${top_builddir}/common/libcommon_ssl.la									\
+	${top_builddir}/mapi4linux/src/libmapi.la								\
 	$(PROG_LIBS)															\
 	$(BOOST_FILESYSTEM_LIBS) $(BOOST_SYSTEM_LIBS) $(ICU_LIBS) $(DL_LIBS)	\
-	$(MYSQL_LIBS) $(SSL_LIBS) $(PAM_LIBS) $(KRB5_LIBS) $(TCMALLOC_LIBS) -lrt
+	$(SSL_LIBS) $(TCMALLOC_LIBS) -lrt
 
 zarafa_server_SOURCES = ECServer.cpp  \
 						ECSoapServerConnection.h ECSoapServerConnection.cpp \
diff -rupN zarafa-7.2.0/provider/soap/Makefile.am zarafa-7.2.0.shared/provider/soap/Makefile.am
--- zarafa-7.2.0/provider/soap/Makefile.am	2014-12-05 09:00:27.000000000 +0100
+++ zarafa-7.2.0.shared/provider/soap/Makefile.am	2015-01-11 22:16:27.174403520 +0100
@@ -1,4 +1,4 @@
-noinst_LTLIBRARIES = libsoapserver.la libsoapclient.la 
+pkglib_LTLIBRARIES = libsoapserver.la libsoapclient.la 
 
 AM_CPPFLAGS = ${ZCPPFLAGS}
 
@@ -6,7 +6,9 @@ AM_CPPFLAGS = ${ZCPPFLAGS}
 #libsoapclient_la_CXXFLAGS = ${AM_CXXFLAGS} -Wno-vla -Wno-sign-compare $(DEBUGFLAGS) $(GSOAP_CFLAGS) -I${top_srcdir}/mapi4linux/include -I${top_srcdir}/provider/include ${GSOAP_CFLAGS}
 
 libsoapserver_la_CXXFLAGS = ${AM_CXXFLAGS} -Wno-sign-compare $(DEBUGFLAGS) $(GSOAP_CFLAGS) -I${top_srcdir}/mapi4linux/include -I${top_srcdir}/provider/include ${GSOAP_CFLAGS}
+libsoapserver_la_LDFLAGS = -release ${PACKAGE_VERSION}
 libsoapclient_la_CXXFLAGS = ${AM_CXXFLAGS} -Wno-sign-compare $(DEBUGFLAGS) $(GSOAP_CFLAGS) -I${top_srcdir}/mapi4linux/include -I${top_srcdir}/provider/include ${GSOAP_CFLAGS}
+libsoapclient_la_LDFLAGS = -release ${PACKAGE_VERSION}
 
 nodist_libsoapserver_la_SOURCES = soapC.cpp soapServer.cpp
 nodist_libsoapclient_la_SOURCES = soapC.cpp soapClient.cpp
diff -rupN zarafa-7.2.0/zarafa-libsync/Makefile.am zarafa-7.2.0.shared/zarafa-libsync/Makefile.am
--- zarafa-7.2.0/zarafa-libsync/Makefile.am	2014-12-05 09:00:26.000000000 +0100
+++ zarafa-7.2.0.shared/zarafa-libsync/Makefile.am	2015-01-11 21:53:11.532757481 +0100
@@ -11,6 +11,8 @@ libzarafasync_la_LIBADD = \
 
 libzarafasync_la_includedir = $(includedir)/libzarafasync
 
+libzarafasync_la_LDFLAGS = -release ${PACKAGE_VERSION}
+
 libzarafasync_la_include_HEADERS = ECLibSync.h ECSync.h ECSyncLog.h	\
 	ECSyncUtil.h ECSyncContext.h ECOfflineABImporter.h ECSyncSettings.h ECResyncSet.h
 
