diff -rupN zarafa-7.2.0/swig/python/zarafa/zarafa/__init__.py zarafa-7.2.0.python-daemon/swig/python/zarafa/zarafa/__init__.py
--- zarafa-7.2.0/swig/python/zarafa/zarafa/__init__.py	2014-12-05 09:00:26.000000000 +0100
+++ zarafa-7.2.0.python-daemon/swig/python/zarafa/zarafa/__init__.py	2015-01-14 06:55:32.210436470 +0100
@@ -49,10 +49,7 @@ from __future__ import with_statement
 
 import contextlib
 import csv
-try:
-    import daemon.pidlockfile
-except ImportError:
-    pass
+import daemon.pidfile
 import datetime
 import grp
 try:
@@ -1800,7 +1797,7 @@ def daemonize(func, options=None, foregr
             if config.get('run_as_group'):
                 gid = grp.getgrnam(config.get('run_as_group')).gr_gid
         if pidfile: # following checks copied from zarafa-ws
-            pidfile = daemon.pidlockfile.TimeoutPIDLockFile(pidfile, 10)
+            pidfile = daemon.pidfile.TimeoutPIDLockFile(pidfile, 10)
             oldpid = pidfile.read_pid()
             if oldpid is None:
                 # there was no pidfile, remove the lock if it's there
