diff -rupN kopanocore-8.3.0.350/installer/linux/Makefile.am kopanocore-8.3.0.350.sysconfig/installer/linux/Makefile.am
--- kopanocore-8.3.0.350/installer/linux/Makefile.am	2016-12-21 09:58:13.000000000 +0100
+++ kopanocore-8.3.0.350.sysconfig/installer/linux/Makefile.am	2016-12-23 12:25:44.230763135 +0100
@@ -58,7 +58,7 @@ suse_sysconfig_path = ${localstatedir}/a
 sles_sysconfig_path = ${localstatedir}/adm/fillup-templates
 sysconf_sysconfigdir = ${@DISTRO@_sysconfig_path}
 sysconf_sysconfig_DATA = sysconfig.txt
-alt_sysconfig_path = ${sysconfdir}/sysconfig
+alt_sysconfig_path = ${sysconfdir}/default
 
 dist_doc_DATA = \
 	db-calc-storesize db-convert-attachments-to-files \
