diff -rupN kopanocore-8.3.0.350/ECtools/search/kopano_search/__init__.py kopanocore-8.3.0.350.python2_7/ECtools/search/kopano_search/__init__.py
--- kopanocore-8.3.0.350/ECtools/search/kopano_search/__init__.py	2016-12-21 09:58:13.000000000 +0100
+++ kopanocore-8.3.0.350.python2_7/ECtools/search/kopano_search/__init__.py	2016-12-30 18:18:07.332798529 +0100
@@ -8,13 +8,9 @@ from multiprocessing import Queue, Value
 import time
 import sys
 
-# Upgrading from Python 2 to Python 3 is not supported
-try:
-    import dbhash
-    from Queue import Empty
-except ImportError:
-    import dbm as dbhash
-    from queue import Empty
+from bsddb3 import db
+
+from Queue import Empty
 
 from kopano_search import plaintext
 import kopano
@@ -80,8 +76,9 @@ CONFIG = {
 
 def db_get(db_path, key):
     """ get value from db file """
-    with closing(dbhash.open(db_path, 'c')) as db:
-        return db.get(key)
+    with closing(db.DB()) as keyDB:
+        keyDB.open(db_path, None, db.DB_HASH, db.DB_CREATE)
+        return keyDB.get(key)
 
 def db_put(db_path, key, value):
     """ store key, value in db file """
@@ -89,6 +86,9 @@ def db_put(db_path, key, value):
         fcntl.flock(lockfile.fileno(), fcntl.LOCK_EX)
         with closing(dbhash.open(db_path, 'c')) as db:
             db[key] = value
+        with closing(db.DB()) as keyDB:
+            keyDB.open(db_path, None, db.DB_HASH, db.DB_CREATE)
+            keyDB.put(key, value)
 
 class SearchWorker(kopano.Worker):
     """ process which handles search requests coming from outlook/webapp, according to our internal protocol """
