diff -rupN kopanocore-8.3.0/ECtools/search/Makefile.am kopanocore-8.3.0.automake/ECtools/search/Makefile.am
--- kopanocore-8.3.0/ECtools/search/Makefile.am	2017-04-26 17:13:49.000000000 +0200
+++ kopanocore-8.3.0.automake/ECtools/search/Makefile.am	2017-04-27 20:32:40.965749958 +0200
@@ -16,7 +16,7 @@ install-exec-local:
 	rm -Rf build
 
 install-data-local:
-	mkdir -p "${DESTDIR}${localstatedir}/lib/kopano/search"
+	mkdir -p "${DESTDIR}${localstatedir}/kopano/search"
 
 uninstall-local:
 	rm -Rf ${DESTDIR}${pythondir}/kopano_search \
diff -rupN kopanocore-8.3.0/installer/linux/Makefile.am kopanocore-8.3.0.automake/installer/linux/Makefile.am
--- kopanocore-8.3.0/installer/linux/Makefile.am	2017-04-26 17:13:49.000000000 +0200
+++ kopanocore-8.3.0.automake/installer/linux/Makefile.am	2017-04-27 20:34:53.713434675 +0200
@@ -48,7 +48,7 @@ dist_sysconf_config_DATA = ${config_file
 sysconf_initdir=${sysconfdir}/init.d
 sysconf_init_SCRIPTS = ${@DISTRO@_initscript_files}
 
-rhel_sysconfig_path = ${sysconfdir}/sysconfig
+rhel_sysconfig_path = ${sysconfdir}/default
 suse_sysconfig_path = ${localstatedir}/adm/fillup-templates
 sles_sysconfig_path = ${localstatedir}/adm/fillup-templates
 sysconf_sysconfigdir = ${@DISTRO@_sysconfig_path}
@@ -89,7 +89,7 @@ EXTRA_DIST = ${BUILT_SOURCES} ldap.m4 \
 
 install-data-hook:
 	install -d -m 755 ${DESTDIR}/${sysconf_configdir}/license
-	install -d -m 755 ${DESTDIR}/${localstatedir}/lib/kopano/report
+	install -d -m 755 ${DESTDIR}/${localstatedir}/kopano/report
 	mv -v ${DESTDIR}${sysconf_rotatedir}/kopano.logrotate ${DESTDIR}${sysconf_rotatedir}/kopano
 	mv -v ${DESTDIR}${sysconf_crondir}/kopano-client-update.cron ${DESTDIR}${sysconf_crondir}/kopano-client-update
 	if test ${DISTRO} = "sles" -o ${DISTRO} = "suse"; then \
diff -rupN kopanocore-8.3.0/spooler/python/Makefile.am kopanocore-8.3.0.automake/spooler/python/Makefile.am
--- kopanocore-8.3.0/spooler/python/Makefile.am	2017-04-26 17:13:49.000000000 +0200
+++ kopanocore-8.3.0.automake/spooler/python/Makefile.am	2017-04-27 20:35:41.698340936 +0200
@@ -9,5 +9,5 @@ kopano_spoolerdir = ${datadir}/kopano-sp
 kopano_spooler_DATA = mapiplugin.py pluginmanager.py plugintemplates.py wraplogger.py zconfig.py zinterval.py zunit.py
 
 install-data-hook:
-	mkdir -p ${DESTDIR}/${localstatedir}/lib/kopano/dagent/plugins/
-	mkdir -p ${DESTDIR}/${localstatedir}/lib/kopano/spooler/plugins/
+	mkdir -p ${DESTDIR}/${localstatedir}/kopano/dagent/plugins/
+	mkdir -p ${DESTDIR}/${localstatedir}/kopano/spooler/plugins/
