diff -rupN kopanocore-8.3.0/ECtools/search/kopano_search/__init__.py kopanocore-8.3.0.python2_7/ECtools/search/kopano_search/__init__.py
--- kopanocore-8.3.0/ECtools/search/kopano_search/__init__.py	2017-04-26 17:13:49.000000000 +0200
+++ kopanocore-8.3.0.python2_7/ECtools/search/kopano_search/__init__.py	2017-04-27 21:58:09.640471897 +0200
@@ -10,7 +10,7 @@ import sys
 
 # Upgrading from Python 2 to Python 3 is not supported
 try:
-    import dbhash
+    from bsddb3 import db
     from Queue import Empty
 except ImportError:
     import dbm as dbhash
@@ -80,15 +80,17 @@ CONFIG = {
 
 def db_get(db_path, key):
     """ get value from db file """
-    with closing(dbhash.open(db_path, 'c')) as db:
-        return db.get(key)
+    with closing(db.DB()) as keyDB:
+        keyDB.open(db_path, None, db.DB_HASH, db.DB_CREATE)
+        return keyDB.get(key)
 
 def db_put(db_path, key, value):
     """ store key, value in db file """
     with open(db_path+'.lock', 'w') as lockfile:
         fcntl.flock(lockfile.fileno(), fcntl.LOCK_EX)
-        with closing(dbhash.open(db_path, 'c')) as db:
-            db[key] = value
+        with closing(db.DB()) as keyDB:
+            keyDB.open(db_path, None, db.DB_HASH, db.DB_CREATE)
+            keyDB.put(key, value)
 
 if sys.hexversion >= 0x03000000:
     def _is_unicode(s):
