diff -ruN ocis-3.0.0-alpha.2/services/idp/Makefile ocis-3.0.0-alpha.2.build/services/idp/Makefile
--- ocis-3.0.0-alpha.2/services/idp/Makefile	2023-04-04 09:10:24.000000000 +0200
+++ ocis-3.0.0-alpha.2.build/services/idp/Makefile	2023-04-06 15:03:34.767589307 +0200
@@ -42,13 +42,15 @@
 
 .PHONY: assets/identifier/static/logo.svg # force overwrite
 assets/identifier/static/logo.svg:
-	curl --fail -o assets/identifier/static/logo.svg ${LOGO_URL}
+#	curl --fail -o assets/identifier/static/logo.svg ${LOGO_URL}
+	cp ../../../logo.svg assets/identifier/static/logo.svg
 
 FAVICON_URL = https://raw.githubusercontent.com/owncloud/assets/main/favicon.ico
 
 .PHONY: assets/identifier/static/favicon.ico # force overwrite
 assets/identifier/static/favicon.ico:
-	curl --fail -o assets/identifier/static/favicon.ico ${FAVICON_URL}
+#	curl --fail -o assets/identifier/static/favicon.ico ${FAVICON_URL}
+	cp ../../../favicon.ico assets/identifier/static/favicon.ico
 
 .PHONY: pnpm-build
 pnpm-build: node_modules
diff -ruN ocis-3.0.0-alpha.2/services/idp/package.json ocis-3.0.0-alpha.2.build/services/idp/package.json
--- ocis-3.0.0-alpha.2/services/idp/package.json	2023-04-04 09:10:24.000000000 +0200
+++ ocis-3.0.0-alpha.2.build/services/idp/package.json	2023-04-06 15:05:31.416763775 +0200
@@ -5,7 +5,7 @@
   "homepage": ".",
   "scripts": {
     "analyze": "source-map-explorer 'build/static/js/*.js'",
-    "build": "node --openssl-legacy-provider scripts/build.js && rm -f build/service-worker.js",
+    "build": "node scripts/build.js --openssl-legacy-provider && rm -f build/service-worker.js",
     "licenses": "NODE_PATH=./node_modules node ../scripts/js-license-ranger.js",
     "licenses:check": "license-checker-rseidelsohn --summary --relativeLicensePath --onlyAllow 'Python-2.0;Apache*;Apache License, Version 2.0;Apache-2.0;Apache 2.0;Artistic-2.0;BSD;BSD-3-Clause;CC-BY-3.0;CC-BY-4.0;CC0-1.0;ISC;MIT;MPL-2.0;Public Domain;Unicode-TOU;Unlicense;WTFPL;ODC-By-1.0'  --excludePackages 'identifier;kpop;po2json;unicoderegexp'",
     "licenses:csv": "license-checker-rseidelsohn --relativeLicensePath --csv --out ../../third-party-licenses/node/idp/third-party-licenses.csv",
diff -ruN ocis-3.0.0-alpha.2/services/web/Makefile ocis-3.0.0-alpha.2.build/services/web/Makefile
--- ocis-3.0.0-alpha.2/services/web/Makefile	2023-04-04 09:10:24.000000000 +0200
+++ ocis-3.0.0-alpha.2.build/services/web/Makefile	2023-04-06 15:04:14.847303678 +0200
@@ -32,8 +32,10 @@
 
 .PHONY: pull-assets
 pull-assets:
-	git clean -xfd assets
-	curl --fail -slL -o- https://github.com/owncloud/web/releases/download/$(WEB_ASSETS_VERSION)/web.tar.gz | tar xzf - -C assets/
+#	git clean -xfd assets
+#	curl --fail -slL -o- https://github.com/owncloud/web/releases/download/$(WEB_ASSETS_VERSION)/web.tar.gz | tar xzf - -C assets/
+	rm -rf assets/*
+	cp -r ../../../web/* assets/
 
 ############ licenses ############
 .PHONY: ci-node-check-licenses
