diff -rupN ocis-3.0.0-alpha.1/services/idp/Makefile ocis-3.0.0-alpha.1.build/services/idp/Makefile
--- ocis-3.0.0-alpha.1/services/idp/Makefile	2023-03-08 11:36:45.000000000 +0100
+++ ocis-3.0.0-alpha.1.build/services/idp/Makefile	2023-03-11 04:29:37.163464625 +0100
@@ -42,13 +42,15 @@ LOGO_URL = https://raw.githubusercontent
 
 .PHONY: assets/identifier/static/logo.svg # force overwrite
 assets/identifier/static/logo.svg:
-	curl --fail -o assets/identifier/static/logo.svg ${LOGO_URL}
+#	curl --fail -o assets/identifier/static/logo.svg ${LOGO_URL}
+	cp ../../../logo.svg assets/identifier/static/logo.svg
 
 FAVICON_URL = https://raw.githubusercontent.com/owncloud/assets/main/favicon.ico
 
 .PHONY: assets/identifier/static/favicon.ico # force overwrite
 assets/identifier/static/favicon.ico:
-	curl --fail -o assets/identifier/static/favicon.ico ${FAVICON_URL}
+#	curl --fail -o assets/identifier/static/favicon.ico ${FAVICON_URL}
+	cp ../../../favicon.ico assets/identifier/static/favicon.ico
 
 .PHONY: pnpm-build
 pnpm-build: node_modules
diff -rupN ocis-3.0.0-alpha.1/services/web/Makefile ocis-3.0.0-alpha.1.build/services/web/Makefile
--- ocis-3.0.0-alpha.1/services/web/Makefile	2023-03-08 11:36:45.000000000 +0100
+++ ocis-3.0.0-alpha.1.build/services/web/Makefile	2023-03-11 03:54:49.514245862 +0100
@@ -32,8 +32,10 @@ ci-node-generate: pull-assets
 
 .PHONY: pull-assets
 pull-assets:
-	git clean -xfd assets
-	curl --fail -slL -o- https://github.com/owncloud/web/releases/download/$(WEB_ASSETS_VERSION)/web.tar.gz | tar xzf - -C assets/
+#	git clean -xfd assets
+#	curl --fail -slL -o- https://github.com/owncloud/web/releases/download/$(WEB_ASSETS_VERSION)/web.tar.gz | tar xzf - -C assets/
+	rm -rf assets/*
+	cp -r ../../../web/* assets/
 
 ############ licenses ############
 .PHONY: ci-node-check-licenses
