diff -rupN php-mobile-webaccess.org/include/login.php php-mobile-webaccess/include/login.php
--- php-mobile-webaccess.org/include/login.php	2010-12-11 19:51:09.576591160 +0100
+++ php-mobile-webaccess/include/login.php	2010-12-11 19:52:21.430340390 +0100
@@ -67,6 +67,11 @@ function login()
 		$_SESSION["password"]= $_POST["password"];
 	}
 	
+	if ((isset($_SERVER["PHP_AUTH_USER"])) && (isset($_SERVER["PHP_AUTH_PW"]))) {
+		$_SESSION["username"]= $_SERVER["PHP_AUTH_USER"];
+		$_SESSION["password"]= $_SERVER["PHP_AUTH_PW"];
+	}
+	
 	if ((isset($_SESSION["username"])) && (isset($_SESSION["password"]))) {
 
 		if ($var = $mapi->logon($_SESSION["username"], $_SESSION["password"], SERVER)) {
diff -rupN php-mobile-webaccess.org/render/html/header.tpl php-mobile-webaccess/render/html/header.tpl
--- php-mobile-webaccess.org/render/html/header.tpl	2010-12-11 19:51:09.466595626 +0100
+++ php-mobile-webaccess/render/html/header.tpl	2010-12-11 19:53:11.391645184 +0100
@@ -1,6 +1,7 @@
 <html>
 <head>
 <title>{$title|escape}</title>
+<meta name="viewport" content="width=300" />
 <link rel="stylesheet" type="text/css" href="css/style.css">
 {literal}
 <script>
