diff -rupN grafana-6.4.0/go.mod grafana-6.4.0.go-1.13/go.mod
--- grafana-6.4.0/go.mod	2019-10-01 11:37:48.000000000 +0200
+++ grafana-6.4.0.go-1.13/go.mod	2019-10-01 22:32:50.679273955 +0200
@@ -72,7 +72,7 @@ require (
 	golang.org/x/net v0.0.0-20190724013045-ca1201d0de80
 	golang.org/x/oauth2 v0.0.0-20190319182350-c85d3e98c914
 	golang.org/x/sync v0.0.0-20190423024810-112230192c58
-	golang.org/x/xerrors v0.0.0-20190410155217-1f06c39b4373
+	golang.org/x/xerrors v0.0.0-20190717185122-a985d3407aa7
 	gopkg.in/alexcesaro/quotedprintable.v3 v3.0.0-20150716171945-2caba252f4dc // indirect
 	gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d // indirect
 	gopkg.in/ini.v1 v1.46.0
